docs/pages/blog/mui-next-js-app-router.md

```diff
@@ -0,0 +1,47 @@
+---
+title: MUI Core libraries support the Next.js App Router
+description: Material UI, Base UI, and Joy UI are now compatible with the App Router as Client Components. Get started using the latest Next.js features with MUI!
+date: 2023-07-18T00:00:00.000Z
+authors: ['samuelsycamore']
+card: true
+tags: ['News']
+---
+
+With [v5.14.0](https://github.com/mui/material-ui/releases/tag/v5.14.0), MUI's Core component libraries—Material UI, Base UI, and Joy UI—are now compatible with the Next.js App Router. 🚀
+
+<img alt="MUI and Next.js logos" src="/static/blog/mui-next-js-app-router/logos.png" loading="lazy" width="2076" height="900" />
+
+## Adapting to the new paradigm
+
+React 18 introduced the concept of React Server Components, and Next.js 13 gave us the first framework implementation of RSCs with the App Router.
```

Comment:
```suggestion
React 18 introduced the concept of React Server Components, and Next.js 13 gave us the most mature framework implementation of RSCs to date with the App Router.
```
I think "most mature" is more notable than "first", what do you think? @samuelsycamore

Reply:
I'm cool with that! 🤝